src/index.html

```diff
@@ -16,7 +16,11 @@
       href="./styles/index.scss"
     />
   </head>
-  <body>
-    <h1>Stop watch</h1>
+  <body class="body">
+    <div class="stopwatch stopwatch--sped-up">
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), class names should be lowercase and if they consist of multiple words, they should be separated by a hyphen. The modifier class `stopwatch--sped-up` should be `stopwatch--sped-up`, which is correct, but ensure all your class names follow this rule.